package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "express": "^4.16.3",
     "ldapjs": "^1.0.2",
-    "smtp-server": "^3.4.5"
+    "smtp-server": "^3.4.6"
```

Comment:
Keep only relevant changes in your PRs if possible

Reply:
Sure